docs/public/content/vuln_db/manuals/kb_importer.md

```diff
@@ -0,0 +1,152 @@
+# KB(Knowledge Base) Importer
+
+KB-Importer is a java application that is used to import vulnerabilities in Steady Backend. 
+
+The vulnerability database contains known vulnerabilities for which a fix exists. In fact, the fix (commit fix in the vulnerable library project) of the vulnerability must be provided in order to add a vulnerability to the database.
```

Comment:
Mmh, I'm afraid this is not true in the new approach: one could just specify affected artifacts and no fix commits actually, as far as I remember...